specs/core/0_beacon-chain.md

```diff
@@ -975,14 +1006,14 @@ def get_attestation_participants(state: BeaconState,
     Returns the participant indices at for the ``attestation_data`` and ``participation_bitfield``.
     """
 
-    # Find the relevant committee
+    # Find the committee in the list with the desired shard
     shard_committees = get_shard_committees_at_slot(state, attestation_data.slot)
-    shard_committee = [x for x in shard_committees if x.shard == attestation_data.shard][0]
-    assert len(participation_bitfield) == ceil_div8(len(shard_committee.committee))
+    shard_committee = [committee for committee, shard in shard_committees if shard == attestation_data.shard][0]
```

Comment:
Suggest we add an assertion before this line to be explicit about handling the case in which the attestation is for a shard that is not found at that slots shuffling

```python
assert attestation.shard in [shard for _, shard in shard_committees]
```